skbeam/core/accumulators/binned_statistic.py

```diff
@@ -434,36 +458,26 @@ def __init__(self, rbins, phibins, rowsize, colsize,
                 will be called on the values in each bin.  Empty bins will be
                 represented by function([]), or NaN if this returns an error.
         """
+        if origin is None:
+            origin = shape[0]//2, shape[1]//2
```

Comment:
I was thinking more for simulations. I generate lots of simulations to predict/confirm certain results and I do play with the parity of pixel number just to test for bugs.

In either case, I can just manually supply the center. I figured I'd ask in case others would agree with this convention.